gravitee-apim-portal-webui/src/app/components/gv-documentation/gv-documentation.component.ts

```diff
@@ -93,24 +92,17 @@ export class GvDocumentationComponent implements AfterViewInit {
     }
   }
 
-  static updateMenuPosition(menuElement, lastTop) {
+  static updateMenuPosition(menuElement) {
     if (menuElement) {
       const scrollTop = document.scrollingElement.scrollTop;
       if (document.querySelector(this.PAGE_COMPONENT)) {
-        const { height } = menuElement.getBoundingClientRect();
         const contentHeight = document.querySelector(this.PAGE_COMPONENT).getBoundingClientRect().height;
-        if (contentHeight - scrollTop <= height) {
-          menuElement.style.top = `${lastTop}px`;
-          menuElement.style.bottom = `${contentHeight - scrollTop}px`;
-          menuElement.style.position = `absolute`;
-          return null;
-        } else {
-          this.reset(menuElement);
-          return scrollTop + ScrollService.getHeaderHeight();
-        }
+
+        menuElement.style['max-height'] = `${Math.max(contentHeight - scrollTop)}px`;
```

Comment:
Weird to use `Math.max` with a single param

Reply:
🤔 🤷‍♂️ 🚮